CommunityToolkit.Mvvm/ComponentModel/Attributes/NotifyRecipientsAttribute.cs

```diff
@@ -33,8 +34,12 @@ namespace CommunityToolkit.Mvvm.ComponentModel;
 ///     }
 /// }
 /// </code>
+/// </para>
+/// <para>
+/// This attribute can also be added to a class, and if so it will affect all generated properties in that type and inherited types.
+/// </para>
 /// </summary>
-[AttributeUsage(AttributeTargets.Field, AllowMultiple = false, Inherited = false)]
-public sealed class AlsoBroadcastChangeAttribute : Attribute
+[AttributeUsage(AttributeTargets.Field | AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+public sealed class NotifyRecipientsAttribute : Attribute
```

Comment:
Are we sure this new name is specific enough to convey that a `PropertyChangedMessage` will be emitted?

Reply:
Mmh...
Maybe `[NotifyPropertyChangedRecipients]`? I mean, a bit verbose but seems consistent, and is self describing.

Reply:
`[NotifyPropertyChangedRecipients]` sounds good. It'll cause a lot less questions and confusion.

Reply:
Done in https://github.com/CommunityToolkit/dotnet/pull/288/commits/bcd9f8267a1ca615db009b6ef6b8bc63f74ae28c 👍